docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: "3"
+
+services:
+  web:
+    build:
+      context: .
+    ports:
+      - target: 4200
+        published: 4200
+    environment:
+      PROTOCOL: 'http:'
+      HOSTNAME: localhost
+      PORT: 4200
+      PATHNAME: /
+      API_ROOT: https://app.stable.retail.backbasecloud.com/api
+      AUTH_URL: https://identity-latest-universal.retail.backbase.eu/auth/realms/backbase
+      AUTH_REALM: backbase
```

Comment:
Realms are a Keycloak specific concept, we don't need to make it explicit when using the full OIDC compliant auth method, this information is embedded in the AUTH_URL

Reply:
removed, thanks